### DISCUSSION TITLE
Issue with acme/autocert

### DISCUSSION THREAD

**Discussion by EthanTuning** [pocketbase/pocketbase#3495] [Q&A] [answered]
I seem to have an issue with serving pocketbase over a custom domain name, that I have set up to point to my server.
This is the error I get when trying to run:

![image](https://github.com/pocketbase/pocketbase/assets/17459105/365f7722-c589-4add-b4bb-6eef371d4179)

Here is my DNS record if that helps at all. But all the IPs and domains all match.

![image](https://github.com/pocketbase/pocketbase/assets/17459105/15872059-2938-4230-9502-60f7f37d3b12)

Do I need to edit some hosts file or something in Debian? I am simply following the "Going to Production" section in the docs.

Thank you for your help!

**Comment by ganigeorgiev** [comment, marked as answer]
Is the domain new and public accessible? 
If it is a new domain the DNS records may need some time to propagate globally (_you can use an online DNS propagation checker to verify it if that's the case_).

**Reply by EthanTuning** [reply]
@ganigeorgiev Thanks for the reply, yes it is a new domain. So yes it may be that it just needs to propagate.

I will look for a tool to check. Thanks again.
